psgo.go

```diff
@@ -231,6 +231,11 @@ var (
 			onHost: true,
 			procFn: processHGROUP,
 		},
+		{
+			normal: "state",
+			header: "STAT",
```

Comment:
I think that's a typo: s/STAT/STATE/

Reply:
`ps` prints  **STAT** as header tag, so I am not sure what's better to use. What do you think?